app/helpers/application_helper/toolbar/ems_datawarehouse_center.rb

```diff
@@ -70,7 +70,9 @@ class ApplicationHelper::Toolbar::EmsDatawarehouseCenter < ApplicationHelper::To
         button(
           :ems_datawarehouse_recheck_auth_status,
           'fa fa-search fa-lg',
-          N_("Re-check Authentication Status for this #{ui_lookup(:table=>'ems_datawarehouse')}"),
+          proc do
+            _("Re-check Authentication Status for this %{warehouse}") % {:warehouse => ui_lookup(:table=>'ems_datawarehouse')}
```

Comment:
Same as above, I'd much rather see `"Re-check Authentication Status for this Datawarehouse Provider"` here.